src/testutils.ts

```diff
@@ -29,13 +31,107 @@ export const TRANSFORMATIONS: {
 
 export const NUM_CHECKS = +(process.env.COPYCAT_NUM_CHECKS || 50)
 
+export const MIN_ENTROPY = 0.98
+
+export const ENTROPY_DATA_SET_SIZE = 1000
+
+export const checkEntropy = <Result>(
```

Comment:
Some extra context: I wanted a way to measure that the generated values are spread out as uniformly as possible. If you imagine a histogram plotted for the values, if all the bars were of similar height for the data set, that would be good. On the other hand, if there is one very large bucket (e.g. if there values are clustering around some point rather than spread out), that would be bad.

Many of the stats tools for this assume a normal distribution or symmetry in the distribution. The best measure I could think of for measuring this was entropy.